src/styles/elements/card.scss

```diff
@@ -0,0 +1,88 @@
+@import "../styles/Stylepack/_paints.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background: $color-white-color;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    max-height: 134px;
+    margin-bottom: 40px;
+    object-fit: contain;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-accent-color;
+  }
+
+  &__product-code {
+    line-height: 14px;
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    color: $secondary-price-color;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-price-color;
+  }
+
+  &__price-money {
+    font-weight: 700;
+    font-size: 16px;
+    color: $main-accent-color;
+  }
+
+  &__buy-link {
+    text-transform: uppercase;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    padding: 14px;
+    font-size: 14px;
+    border-radius: 5px;
+    background-color: $giganiga-blue-color;
+    text-decoration: none;
+    color: $color-white-color;
+    font-weight: 700;
+    height: 40px;
+  }
+
+  &__buy-link:hover {
+    background-color: $color-white-color;
+    color: $giganiga-blue-color;
+    border: 1px solid $giganiga-blue-color;
```

Comment:
Better add transparent border for default state of element and only change its color on hover